src/scrapers/game_details_scrape.py

```diff
@@ -0,0 +1,303 @@
+import requests
+from bs4 import BeautifulSoup
+
+def fetch_page(url):
+    response = requests.get(url)
+    return BeautifulSoup(response.text, 'html.parser')
+
+def extract_teams_and_scores(box_score_section, sport):
+    score_table = box_score_section.find('table', class_='sidearm-table')
+    team_names = []
+    period_scores = []
+
+    for row in score_table.find('tbody').find_all('tr'):
+        if sport == 'ice hockey':
+            team_name_cell = row.find('th')
+            if team_name_cell:
+                team_name = team_name_cell.text.strip().replace("Winner", "").strip()
+                team_name = ' '.join(team_name.split())
+            else:
+                team_name = "Unknown"
+        else:
+            team_name_cell = row.find('td')
+            if team_name_cell:
+                if sport == 'baseball':
+                    team_name = ' '.join(span.text.strip() for span in team_name_cell.find_all('span', class_='hide-on-large-down'))
```

Comment:
I would recommend moving all of the HTML tags to constants.py and using those constants instead of explicitly writing them in the file. You can look at my branch for reference